bin/build/src/i18n/common.clj

```diff
@@ -40,7 +40,9 @@
   (when-let [^Message message (.locateHeader (catalog locale))]
     (let [header-fields (HeaderFields/wrap (.getMsgstr message))]
       (into {} (for [^String k (.getKeys header-fields)]
-                 [k (.getValue header-fields k)])))))
+                 [k (if (= k "Language")
+                      locale
+                      (.getValue header-fields k))])))))
```

Comment:
crowdin was using 4-letter codes here and it was breaking a bunch of downstream assumptions.